### PR TITLE
change to external/msf_gif.h for rendering to gif

### PR DESCRIPTION
**Pull request by chriscamacho**
hmm, suspiciously easy! does seem faster!

### CODE REVIEW

File: src/core.c [hunk 1]
```diff
@@ -147,11 +147,8 @@
 #endif
 
 #if defined(SUPPORT_GIF_RECORDING)
-    #define RGIF_MALLOC RL_MALLOC
-    #define RGIF_FREE RL_FREE
-
-    #define RGIF_IMPLEMENTATION
-    #include "external/rgif.h"      // Support GIF recording
+    #define MSF_GIF_IMPL
```
Comment (raysan5) [inline src/core.c:150]: Required:
```c
#define MSF_GIF_MALLOC RL_MALLOC
#define MSF_GIF_FREE RL_FREE
```

**Comment by raysan5** [comment]
@chriscamacho I added one review, `MSF_GIF_MALLOC`, `MSF_GIF_FREE` macros are required.

**Comment by raysan5** [comment]
@chriscamacho `msf_gif.h` has changed to support memory version as per https://github.com/notnullnotvoid/msf_gif/issues/2 and https://github.com/notnullnotvoid/msf_gif/issues/3. I'm closing this PR for now, it should be re-implemented with the new API.

**Comment by chriscamacho** [comment]
okay not sure if I'll get chance to get to this...

**Pull request closed without merging**